### PR TITLE
update travis config

### PR DESCRIPTION
**Pull request by yvaucher**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
- - travis_install_nightly 7.0
- - pip install coveralls flake8
- - pip install PyPDF2==1.18
- - pip install suds-jurko==0.6
- - git clone https://github.com/OCA/stock-logistics-workflow -b 7.0 $HOME/stock-logistics-workflow
- - git clone https://github.com/OCA/webkit-tools -b 7.0 $HOME/webkit-tools
+  - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
+  - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
+  - travis_install_nightly
+  - pip install coveralls flake8
```
Comment (bwrsandman) [inline .travis.yml]: coveralls and flake are already installed by `travis_install_nightly` you don't need this line

**Comment by yvaucher** [comment]
@bwrsandman thanks
fixed

File: .travis.yml [hunk 2]
```diff
@@ -1,23 +1,27 @@
...
- - pip install PyPDF2==1.18
- - pip install suds-jurko==0.6
- - git clone https://github.com/OCA/stock-logistics-workflow -b 7.0 $HOME/stock-logistics-workflow
- - git clone https://github.com/OCA/webkit-tools -b 7.0 $HOME/webkit-tools
+  - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
+  - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
+  - travis_install_nightly
+  - pip install PyPDF2==1.18
+  - pip install suds-jurko==0.6
+  - git clone https://github.com/OCA/stock-logistics-workflow -b 7.0 ${HOME}/stock-logistics-workflow
```
Comment (pedrobaeza) [inline .travis.yml]: Instead of 7.0, use ${VERSION}

File: .travis.yml [hunk 3]
```diff
@@ -1,23 +1,27 @@
...
- - travis_install_nightly 7.0
- - pip install coveralls flake8
- - pip install PyPDF2==1.18
- - pip install suds-jurko==0.6
- - git clone https://github.com/OCA/stock-logistics-workflow -b 7.0 $HOME/stock-logistics-workflow
- - git clone https://github.com/OCA/webkit-tools -b 7.0 $HOME/webkit-tools
+  - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
+  - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
+  - travis_install_nightly
+  - pip install PyPDF2==1.18
```
Comment (bwrsandman) [inline .travis.yml:17]: Why do you freeze the version here?

What's wrong with https://github.com/OCA/maintainer-quality-tools/blob/master/travis/requirements.txt#L19

File: .travis.yml (hunk 2 above)
Comment (bwrsandman) [inline .travis.yml]: I like this format:

```
- git clone https://github.com/OCA/stock-logistics-workflow ${HOME}/stock-logistics-workflow -b ${VERSION}
```

Easier to read

File: .travis.yml (hunk 3 above)
Comment (yvaucher) [inline .travis.yml:17]: Isn't it better to freeze version for tests?
The chosen version are versions I'm sure it works with.

PyPDF2 is a requirement of delivery_carrier_label_dispatch module

Comment (pedrobaeza) [inline .travis.yml:17]: IMHO, better to try with the last one, and see if it fails

Comment (bwrsandman) [inline .travis.yml:17]: Probably would be good to catch errors if there are version incompatibilities.

I don't like the idea of using a different version than all other repos.

Can you take pypdf out if it is not absolutely necessary

Comment (yvaucher) [inline .travis.yml:17]: Last time I checked, PyPDF was no longer maintained.

PyPDF2 should have some more features like for merging PDF. PyPDF2 is also used in reporting_engine for module base_report_assembler.

I would prefer to keep it.

Comment (bwrsandman) [inline .travis.yml:17]: > Last time I checked, PyPDF was no longer maintained.

Really??? :confounded: It's now a core odoo 8.0 dependency...

Comment (yvaucher) [inline .travis.yml:17]: http://pybrary.net/pyPdf/

Is there a newer than 1.13 version of pyPDF?

Comment (bwrsandman) [inline .travis.yml:17]: Seems not

**Comment by bwrsandman** [comment]
:+1: 

Whoever merges this, please create an issue to fix PEP8 on 7.0

**Comment by yvaucher** [comment]
@bwrsandman already there https://github.com/OCA/carrier-delivery/pull/21

**Pull request merged**